libs/pages/services/src/lib/ui/table-row-services/table-row-services.tsx

```diff
@@ -108,25 +116,34 @@ export function TableRowServices(props: TableRowServicesProps) {
           <Skeleton className="w-full" show={isLoading} width={160} height={16}>
             <div className="w-full flex gap-2 items-center -mt-[1px]">
               {type === ServiceTypeEnum.APPLICATION && (
```

Comment:
Because with just these conditions we have to make the effort everytime to see if you test if it's an gitapp, a container app, etc... Frontend joining us could be lost quickly and so could we after many months